components/Tracking.js

```diff
@@ -1,20 +1,10 @@
 import React, { useEffect } from "react";
 import ReactGA from "react-ga"
 import PropTypes from "prop-types"
-import { getCookiesFromLocalStorage } from 'utils';
+import { trackPageView } from 'utils';
```

Comment:
I extracted this into a utility method so we could hypothetically reuse it if we needed it.